boards/nxp/mimxrt1180_evk/mimxrt1180_evk.dtsi

```diff
@@ -139,3 +140,17 @@
 	pinctrl-0 = <&pinmux_flexpwm2>;
 	pinctrl-names = "default";
 };
+
+&usdhc1 {
+	status = "okay";
+	detect-dat3;
+	pwr-gpios = <&gpio4 14 GPIO_ACTIVE_LOW>;
```

Comment:
according to the schematic should it be active high?

Reply:
Thank you for reminding. 'no-1-8-v' have been added into dts.